styles.css

```diff
@@ -659,139 +659,128 @@ textarea {
 
   input,
   textarea {
-    width: 500px;
+    width: 720px;
   }
+}
 
-  @keyframes scaled {
-    50% {
-      transform: scale(0.4);
-      transform-origin: left;
-      box-shadow: 4px 3px 2px black;
-    }
+/* mobile popup section */
+.popup-container {
+  background-color: silver;
+  position: fixed;
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+  overflow-y: auto;
+}
+
+.popup-contents {
+  background-color: #fff;
+  width: 90%;
+  height: auto;
+  margin: 100px auto;
+  box-sizing: border-box;
+  border-radius: 16px;
+  padding: 20px;
+}
+
+.close-popup {
+  position: absolute;
+  top: 120px;
+  right: 35px;
+  font-size: 20px;
+  cursor: pointer;
+  font-weight: 500;
+  color: silver;
+}
+
+.popup-title {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 2.1em;
+  line-height: 44px;
+  width: 65%;
+  color: #172b4d;
+  margin: 3px 0;
+}
+
+.popup-img {
+  width: 100%;
+  height: auto;
+}
```

Comment:
- [ ] [OPTIONAL] For the responsiveness of your popup window instead of using the `height:auto` I would suggest using `aspect-ratio:2/1` property, as currently the image is almost scaling up to the entire view height which is not a great user experience. You can also make use of `object-fit`  property for the high quality display of the image.